src/hooks/useHandleEvent.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { useIsClient } from './useIsClient';
+
+const EventTypes = {
+  NAVIGATE_PREVIOUS_PAGE: 'popstate',
+};
+
+const useHandleEvent = (eventsMap = {}) => {
```

Comment:
This is a hook I added, we can use in the future, but isn't related to this PR specifically

Reply:
Cool 👍